cli/src/main.rs

```diff
@@ -39,6 +39,8 @@ const CIRCUIT_CREATE_AFTER_HELP: &str = r"DETAILS:
     One or more nodes must be specified using the --node and/or --node-file arguments. These
     arguments can be used on their own or together, but at least one of them is required.
 
+    If the --node argument is specified, 'endpoint' is optional if node alias has been set.
```

Comment:
Could be a little clearer: `..., the 'endpoint' portion of the argument is optional if an alias for the specified node has been set using the 'splinter node alias' command.`

Reply:
Getting rid of this sentence, per @vaporos's comment below